optional/cloudfront_cdn/main.tf

```diff
@@ -0,0 +1,70 @@
+resource "aws_cloudfront_distribution" "cloudfront_distribution" {
+  comment = lower(
+    join("-", [
+    var.client_shortname, var.environment, var.service_name
+    ])
+  )
+
+  origin {
+    connection_attempts = 3
+    connection_timeout  = 10
+    domain_name         = var.origin_domain
+    origin_id           = var.origin_domain
+
+    custom_origin_config {
+      http_port                = 80
+      https_port               = 443
+      origin_keepalive_timeout = 5
+      origin_protocol_policy   = "match-viewer"
+      origin_read_timeout      = 30
+      origin_ssl_protocols     = [
+        "TLSv1",
+        "TLSv1.1",
+        "TLSv1.2",
+      ]
+    }
+  }
+
+
+  enabled             = true
+  is_ipv6_enabled     = true
+
+  # Cache behavior with precedence 1
+  default_cache_behavior {
+    allowed_methods  = ["GET", "HEAD", "OPTIONS"]
+    cached_methods   = ["GET", "HEAD"]
+    target_origin_id = var.origin_domain
+
+    default_ttl            = var.cache_expiration
+    max_ttl                = var.cache_expiration
+    compress               = true
+    viewer_protocol_policy = "allow-all"
+
+    forwarded_values {
+    headers = [
+      "*",
+    ]
+    query_string = true
+    query_string_cache_keys = []
```

Comment:
Doesn't specifying the empty list [disable](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/cloudfront_distribution#query_string_cache_keys) `query_string` caching? The docs mention it needs to be set up to `true` to cache everything - otherwise, only the listed query strings are cached.

Reply:
Thanks for catching it, I've removed this argument and let `query_string` set to true.